src/components/SummaryCard/SummaryCardSelect/__tests__/SummaryCardSelect.spec.js

```diff
@@ -4,17 +4,17 @@
  */
 
 import { render } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
 import React from 'react';
 
 import { SummaryCardSelect } from '../../../..';
 
-import props from '../_mocks_';
-
 describe('SummaryCardSelect', () => {
-  it('renders', () => {
-    expect(
-      render(<SummaryCardSelect className="className" {...props} />).container
-        .firstChild
-    ).toMatchSnapshot();
+  test('should be checked when clicked', () => {
```

Comment:
I don't think this hurts to check but it is technically redundant due to the tests for `SummaryCardContainer` 🤔

Reply:
Good point - in the case of the `SummaryCardContainer`, we're testing the internal logic / negotiation of selected checkboxes. As we pass through our props to the `Checkbox` component this should also be covered by their unit tests. 🤞

I think it's also beneficial to ensure the appropriate `className` is being applied here, and an example of an `...other` prop, as these are behaviors based on consumer input.

Reply:
Good point -- it looks like `id` and `labelText` props I'm using here (which I believe they are required in the `Checkbox` anyway) are passed through the spread attribute in `SummaryCardSelect` (so they are already checking `...other`)

I'll add a test for a custom class 👍